packages/sku/config/babel/babelConfig.js

```diff
@@ -79,6 +79,10 @@ module.exports = ({
           {
             isTSX: true,
             allExtensions: true,
+            // babel equivalent of tsconfig `verbatimModuleSyntax: true`
+            onlyRemoveTypeImports: true,
+            // babel equivalent of tsconfig `allowImportingTsExtension: true`
+            rewriteImportExtensions: true,
```

Comment:
We already enable `allowImportingTsExtensions`, so we should enable it in our babel config too, though it doesn't really affect output code since it's all bundled anyway.